src/Ersatz/Relation/Data.hs

```diff
@@ -35,9 +58,15 @@ relation bnd = do
             return ( p, x )
     return $ build bnd pairs
 
+-- | Constructs an indeterminate relation \( R \subseteq A \times A \) with the constraint, 
+-- that it is symmetric, i.e., \( \forall x, y \in A: ((x,y) \in R) \rightarrow ((y,x) \in R) \).
+--
+-- It can be thought of a symmetric relation as an undirected graph.
 symmetric_relation ::
   (MonadSAT s m, Ix b) =>
-  ((b, b), (b, b)) -> m (Relation b b)
+  ((b, b), (b, b)) -- ^ Since a symmetric relation must be homogeneous, the domain must equals the codomain, 
+                   -- hence for the bounds @((p,q),(r,s))@ must hold @p=q@ and @r=s@.
```

Comment:
```suggestion
  ((b, b), (b, b)) -- ^ Since a symmetric relation must be homogeneous, the domain must equal the codomain. 
                   -- Therefore, given bounds @((p,q),(r,s))@, it must hold that @p=q@ and @r=s@.
```